packages/react/src/patterns/blocks/ContentGroupPictograms/__stories__/index.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../../../../../styles/scss/patterns/blocks/content-group-pictograms';
```

Comment:
Aren't we already importing everything? For some reason, I was thinking @kennylam changed storybook to just import everything. Feel free to correct me if I'm wrong though.

Reply:
we're not importing globally. We are importing this way with the story level scss file that includes all of the scss files we need to import, so this is ok.